packages/api/portfolioDrafts/submit/submitPortfolioDraft.ts

```diff
@@ -13,7 +13,10 @@ import { PortfolioDraft } from "../../models/PortfolioDraft";
 import { v4 as uuidv4 } from "uuid";
 import { ProvisioningStatus } from "../../models/ProvisioningStatus";
 import { dynamodbDocumentClient as client } from "../../utils/dynamodb";
+import { SendMessageCommand } from "@aws-sdk/client-sqs";
+import { sqsClient } from "../../utils/sqs";
 const TABLE_NAME = process.env.ATAT_TABLE_NAME ?? "";
+const QUEUE_URL = process.env.ATAT_QUEUE_URL ?? "";
```

Comment:
As another tech debt item, I'd love to think through a better way to handle these envvar names. It really feels like they should be determined and exported via `utils/{sqs,dynamodb}.ts` so that we can change the logic of what the table name is or how it's determined without having to change every file.

In the future, we may want to put this value in Systems Manager Parameter Store.